spec/features/police_dashboard_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'feature_helper'
+
+RSpec.feature 'police dashboard', type: :feature do
+  scenario 'login as police user' do
+    banbury_police_station = create(:police_custody, name: 'Banbury Police Station')
+    cardigan_police_station = create(:police_custody, name: 'Cardigan Police Station')
+    croydon_custody_unit = create(:police_custody, name: 'Croydon Custody Unit')
+
+    escort_1 = create(:escort, :issued, :completed, prison_number: 'A9876XC')
+    escort_1.move.update(date: Date.today, from_establishment: banbury_police_station)
+    escort_1.detainee.update(forenames: 'PETER', surname: 'GABRIEL')
+
+    escort_2 = create(:escort, :issued, :completed)
+    escort_2.move.update(date: Date.today, from_establishment: cardigan_police_station)
+
+    escort_3 = create(:escort, :completed)
+    escort_3.move.update(date: Date.tomorrow, from_establishment: banbury_police_station)
```

Comment:
I wonder if we could improve the factories so we could avoid this updating to tweak the move & detainee info - maybe some new traits...?